extensions/amp-story-shopping/validator-amp-story-shopping.protoascii

```diff
@@ -19,24 +19,52 @@ html_format: AMP
   amp_layout: {
     supported_layouts: CONTAINER
   }
+  attrs: {
+    name: "data-product-id"
+    mandatory: true
+  }
 }
-tags: {  # <amp-story-shopping-config>
+tags: {  # <amp-story-shopping-attachment>
```

Comment:
Please add `mandatory_last_child: true`.
This follows the pattern of page attachment.
https://github.com/ampproject/amphtml/blob/main/extensions/amp-story/validator-amp-story.protoascii#L843-L846